### DISCUSSION TITLE
Fall 2022 League

### DISCUSSION THREAD

**Discussion by bevns** [es-na-battlesnake/snakes#233] [General]
No official announcement has been made just yet but it looks like the Fall season is slated to kick off later this month: https://docs.battlesnake.com/guides/playing-battlesnake/leagues#when-are-leagues-happening

Opening this discussion to start a place for us to decide our approach to the Fall league.

**Comment by bevns** [comment]
I am interested in a collaborative team snake again.

**Reply by djdefi** [reply]
Same, I'd like to iterate on one of the existing ones.
